enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/tx/BatchingTxApplier.java

```diff
@@ -53,12 +55,13 @@
     private volatile boolean stopped;
 
     public BatchingTxApplier( int maxBatchSize, Supplier<TransactionIdStore> txIdStoreSupplier,
-            Supplier<TransactionCommitProcess> commitProcessSupplier,
-            Monitors monitors, LogProvider logProvider )
+                              Supplier<TransactionCommitProcess> commitProcessSupplier, Monitors monitors,
+                              PageCursorTracerSupplier pageCursorTracerSupplier, LogProvider logProvider )
```

Comment:
This indentation looks non-standard (http://neo4j.github.io/).

Reply:
Looks pretty standard according to your linked style.